crates/sui-network/src/state_sync/mod.rs

```diff
@@ -127,6 +128,10 @@ impl Handle {
     pub fn subscribe_to_synced_checkpoints(&self) -> broadcast::Receiver<VerifiedCheckpoint> {
         self.checkpoint_event_sender.subscribe()
     }
+
+    pub fn get_sender(&self) -> &broadcast::Sender<VerifiedCheckpoint> {
+        &self.checkpoint_event_sender
+    }
```

Comment:
Lets remove this so that it isn't exposed and unintentionally allow for other components other than state-sync to send messages to the broadcast channel